tests/Support/SupportCollectionTest.php

```diff
@@ -5590,14 +5590,12 @@ public function testPercentageReturnsNullForEmptyCollections($collection)
     /**
      * Provides each collection class, respectively.
      *
-     * @return array
+     * @return \Generator
      */
-    public static function collectionClassProvider()
+    public static function collectionClassProvider(): \Generator
     {
-        return [
-            [Collection::class],
-            [LazyCollection::class],
-        ];
+        yield [Collection::class];
+        yield [LazyCollection::class];
```

Comment:
Is this change related to defer? oO

Reply:
Not really, it's just a minor optimization.

Reply:
Not sure if that would be better suited for a separate PR 🤔

Reply:
I will rollback it!